cmd/root.go

```diff
@@ -43,6 +44,13 @@ var rootCmd = &cobra.Command{
 	Long: `After installation go to your project directory
 and execute the following command:
 lefthook install`,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		var fs = afero.NewOsFs()
+		gitInitialized, _ := afero.DirExists(fs, filepath.Join(getRootPath(), ".git"))
+		if !gitInitialized {
+			log.Fatal("git is not initialized")
```

Comment:
That message should be more verbose to give user a hint what to do. What `git is not initialized` does mean, after all?
Maybe something like that?
```suggestion
			log.Fatal("This command must be executed within git repository. Change working directory or initialize new repository with `git init`.")
```

Reply:
✅